.travis.yml

```diff
@@ -17,4 +17,13 @@ notifications:
     recipients:
       - ajax@bynder.com
     on_success: never
-    on_failure: always
\ No newline at end of file
+    on_failure: always
+
+deploy:
```

Comment:
So far we've been releasing using `python setup.py bdist_wheel`, I think you need to specify `distributions: "bdist_wheel"`

Reply:
It's in the Makefile for example: https://github.com/Bynder/bynder-python-sdk/blob/master/Makefile#L25